fvm/evm/stdlib/contract.cdc

```diff
@@ -34,6 +34,8 @@ contract EVM {
         receiptRoot: [UInt8; 32],
         // root hash of all the transaction hashes
         transactionHashRoot: [UInt8; 32],
+        /// value returned for PREVRANDAO opcode
+        prevrandao: [UInt8; 32],
```

Comment:
This is not a valid contract update.

We can deploy this, but its tricky. We have to remove a safety mechanism in cadence that disallows deleting contracts that declare enums, then remove the contract and redeploy the updated version back. 

We did this a few times on previewnet, but we have not done it on testnet yet I think.

Reply:
ah I see, do we have a chance to make it possible or we need to revisit contract changes ?